### PR TITLE
fix: relay openapi definitions

### PR DESCRIPTION
**Pull request by SionoiS**
# Description
Forgot to update openapi.yml for the new autosharding relay API in #1983

I also removed the topic when auto posting as it's already included in the message itself.

### CODE REVIEW

File: waku/waku_api/rest/relay/openapi.yaml [hunk 1]
```diff
@@ -106,6 +106,100 @@ paths:
...
+      summary: Subscribe a node to an array of topics
+      description: Subscribe a node to an array of Content topics.
+      operationId: postSubscriptions
+      tags:
+        - relay
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RelayPostSubscriptionsRequest'
```
Comment (NagyZoltanPeter) [inline waku/waku_api/rest/relay/openapi.yaml:166]: I think it shall be changed here having a RelayPostAutoSubscriptionRequest type, with array of contentTopic
Or just use here array of contentTopic without schema type.

File: waku/waku_api/rest/relay/openapi.yaml [hunk 2]
```diff
@@ -106,6 +106,100 @@ paths:
...
+      summary: Unsubscribe a node from an array of topics
+      description: Unsubscribe a node from an array of Content topics.
+      operationId: deleteSubscriptions
+      tags:
+        - relay
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RelayDeleteSubscriptionsRequest'
```
Comment (NagyZoltanPeter) [inline waku/waku_api/rest/relay/openapi.yaml:189]: Same as with Post subscription request.

File: waku/waku_api/rest/relay/openapi.yaml (hunk 1 above)
Comment (SionoiS) [reply waku/waku_api/rest/relay/openapi.yaml:166]: `RelayPostSubscriptionsRequest` are string types so both pubsub and content topic works no?

File: waku/waku_api/rest/relay/openapi.yaml [hunk 3]
```diff
@@ -72,15 +74,18 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/RelayPostSubscriptionsRequest'
+              type array:
+              items:
+                $ref: '#/components/schemas/ContentTopic'
```
Comment (NagyZoltanPeter) [inline waku/waku_api/rest/relay/openapi.yaml:79]:
```suggestion
                $ref: '#/components/schemas/PubsubTopic'
```

File: waku/waku_api/rest/relay/openapi.yaml [hunk 4]
```diff
@@ -94,18 +99,119 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/RelayDeleteSubscriptionsRequest'
+              type array:
+              items:
+                $ref: '#/components/schemas/ContentTopic'
```
Comment (NagyZoltanPeter) [inline waku/waku_api/rest/relay/openapi.yaml:104]:
```suggestion
                $ref: '#/components/schemas/PubsubTopic'
```

**Review by NagyZoltanPeter — CHANGES_REQUESTED** [review]
After all its all good, except those two typo.

**Comment by SionoiS** [comment]
I hate it when I have no tool to type check everything!

**Review by NagyZoltanPeter — APPROVED** [review]
Super cool! Thank you and for your patience on my late request!
Its unfortunate such a small change requires that much code to be changed.

**Pull request merged**